apps/meteor/app/push/server/fcm.ts

```diff
@@ -81,7 +86,7 @@ async function fetchWithRetry(url: string, _removeToken: () => void, options: Ex
 	const retryAfter = response.headers.get('retry-after');
 	const retryAfterSeconds = retryAfter ? parseInt(retryAfter, 10) : 60;
 
-	if (response.status === 404) {
+	if (response.status === 404 || response.status === SENDER_ID_MISMATCH_ERROR_CODE) {
```

Comment:
Why not use an includes here and convert the 404 to a constant too?

Reply:
Didn't do it because 404 is one of those numbers that everyone knows what means, while 403 might not make a lot of sense of why it needs to have the token removed.

But I guess there is no problem on having both as constants :)